vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,167 @@ lyt3 <- basic_table() %>%
 tbl3 <- build_table(lyt3, ex_adsl)
 tbl3
 ```
+
+### Top-left Materials
+
+The sequence of sequence of strings printed in the area between the column header display and the first row label (\emph{top left material}) can be modified using the `top_left()` function.
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("SEX") %>%
+  analyze(c("AGE", "STRATA1"), function(x) {
+    if (is.numeric(x)) {
+      in_rows(
+        "mean" = rcell(mean(x)),
+        "sd" = rcell(sd(x)),
+        .formats = c("xx.x"), .aligns = "left"
+      )
+    } else if (is.factor(x)) {
+      rcell(length(unique(x)), align = "right")
+    } else {
+      stop("Unsupported type")
+    }
+  }, show_labels = "visible", na_str = "NE") %>%
+  build_table(ex_adsl)
+
+lyt
+
+# Adding top-left material
+top_left(lyt) <- "New top-left material here"
+
+lyt
+```
+
+
+### Table Inset
+
+Table title, table body, referential footnotes and and main footers can be inset from the left alignment of the titles and provenance footer materials.
+
+Below is a simplified example:
+
+```{r}
+lyt <- basic_table() %>%
+  analyze("AGE")
+
+# without inset
+tbl <- build_table(lyt, DM)
+tbl
+
+# with inset of 3 characters
+table_inset(tbl) <- 3
+tbl
+```
+
+Below is an example with a table produced for clinical data. Compare the inset of the table and main footer.
+
+```{r}
+lyt <- basic_table(
+  title = "Study XXXXXXXX",
+  subtitles = c("subtitle YYYYYYYYYY", "subtitle2 ZZZZZZZZZ"),
+  main_footer = "Analysis was done using cool methods that are correct",
+  prov_footer = "file: /path/to/stuff/that/lives/there HASH:1ac41b242a"
+) %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("SEX", page_by = TRUE, page_prefix = "Patient Subset - Gender", split_fun = drop_split_levels) %>%
+  split_rows_by("STRATA1") %>%
+  analyze("AGE", mean, format = "xx.x")
+
+# without inset
+tbl <- build_table(lyt, DM)
+tbl
+
+
+# with inset
+table_inset(tbl) <- 5
+tbl
+```
+
+
+### Horizontal Separation
+A character value can be specified to modify the horizontal separation between column headers and the table. Below, we replace the default line with "=".
+```{r}
+tbl <- basic_table() %>%
+        split_cols_by("Species") %>%
+        add_colcounts() %>%
+        analyze(c("Sepal.Length", "Petal.Width"), function(x) {
+            in_rows(
+                mean_sd = c(mean(x), sd(x)),
+                var = var(x),
+                min_max = range(x),
+                .formats = c("xx.xx (xx.xx)", "xx.xxx", "xx.x - xx.x"),
+                .labels = c("Mean (sd)", "Variance", "Min - Max")
+            )
+        }) %>%
+        build_table(iris, hsep = "=")
+tbl
+```
+
+### Section Dividers
+
+A character value can be specified as a section divider which succeed every group defined by a split instruction. Below, a "+" is repeated and used as a section divider.
```

Comment:
Please add an example with `" "`, which generates a blank line, which will be the most common usage, but also leave in where we set it to something other than that so that people know they can.

Also, please include an example with multiple levels of row splitting that have different section dividers, and call out explicitly the behavior that only one will be printed if they would "pile up" right next to eachother, and that a trailing divider at the end of the table is never printed.